src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (number) => {
+      calculator.result += number;
```

Comment:
Instead of using 'calculator.result', use 'this.result' to reference the object property. Check all methods

Reply:
Not resolved.